Makefile

```diff
@@ -461,4 +461,13 @@ test-all:
 # run all tests
 .PHONY: test-integration
 test-integration:
-	go test -coverpkg=./... -v -race -covermode atomic -coverprofile=coverage.out -tags=integration ./test/integration...
\ No newline at end of file
+	go test -coverpkg=./... -v -race -covermode atomic -coverprofile=coverage.out -tags=integration ./test/integration...
+
+.PHONY: test-cyclonus
+test-cyclonus:
+	cd test/cyclonus && bash ./test-cyclonus.sh
+	cd ..
+
+.PHONY: kind
```

Comment:
Do we need this? It seems kind cluster is created from cyclonus-netpol-test.yaml file.

Reply:
This is more for developer consistency, I have a script on my devbox to accomplish this, but I found it useful to have in the makefile more than once. I can remove if there are strong feelings

Reply:
Good to have consistency.